common/lib/common-definitions/package.json

```diff
@@ -62,18 +109,20 @@
 		"eslint-config-prettier": "~9.0.0",
 		"prettier": "~3.0.3",
 		"rimraf": "^2.6.2",
-		"tsc-multi": "^1.1.0",
 		"typescript": "~5.1.6"
 	},
 	"packageManager": "pnpm@7.32.3+sha512.c28d73f0d82a6c4712a40b91aa43d7397e119b16f35e5e75fe9b7e8fd4f2fc6dfbb68bb3ffac3473a958bbafa7888b79ec384ad122537378089a3a2a19b9900e",
 	"fluidBuild": {
-		"branchReleaseTypes": {
-			"main": "minor",
-			"release/**": "patch"
-		},
 		"tasks": {
```

Comment:
Is this okay to remove?

Reply:
Yeah, that's stuff from old build-tools versions that's no longer used.